### PR TITLE
DATA-4405 Added tox

### PR DESCRIPTION
**Pull request by Bakuutin**

### CODE REVIEW

**Comment by Bakuutin** [comment]
@theorchard/data-eng

**Comment by Bakuutin** [comment]
@someboredkiddo, should I open that PR here, or in the original repo?

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py27,py34,py35,py36
+
+[testenv]
+deps=pytest
+commands=py.test
+
+[testenv:py27]
+deps = -rrequirements-py2.txt
```
Comment (scotteggs) [inline tox.ini:9]: looks like a double r in rrequirements-py2.txt

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+[testenv:py3]
+deps = -rrequirements.txt
```
Comment (scotteggs) [inline tox.ini:12]: here too rrequirements

**Comment by someboredkiddo** [comment]
@Bakuutin I would merge here first and then go back to the https://github.com/xethorn/garcon

Ortalie (xethorn) is pretty busy and it would take awhile to get it merged in the original repo

File: tox.ini (hunk 1 above)
Comment (Bakuutin) [reply tox.ini:9]: It's a flag `-r` plus `requirements-py2.txt`
http://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints

**Comment by scotteggs** [comment]
R+ pending tests passing.

**Review by scotteggs — APPROVED** [review]

**Pull request merged**